dist/index.js

```diff
@@ -31173,7 +31173,9 @@ done`;
         yield execShellCommand(["vcs import --force --recursive src/ < package.repo"], options);
         // Print HEAD commits of all repos
         yield execShellCommand(["vcs log -l1 src/"], options);
-        if (isLinux) {
+        // rosdep does not really work on Windows, so do not use it
+        // See: https://github.com/ros-infrastructure/rosdep/issues/610
+        if (!isWindows && !skipRosdepInstall) {
```

Comment:
if would keep the `if (isLinux)` before the `apt update`/`dnf check-update` part, since this could technically be non-Window, non-Linux (even though it's probably never the case in practice): https://nodejs.org/docs/latest-v20.x/api/process.html#processplatform.

Reply:
fixing now :)